tests/main/interfaces-process-control/task.yaml

```diff
@@ -40,11 +40,10 @@ execute: |
     echo "Then the snap is not able to kill an existing process"
     sleep 5m &
     pid=$!
+    tests.cleanup defer "kill $pid 2>/dev/null || true"
+
     if process-control-consumer.signal SIGTERM "$pid" 2> process-kill.error; then
         echo "Expected permission error accessing killing a process with disconnected plug"
         exit 1
     fi
-    grep -q "Permission denied" process-kill.error
-    kill -s 0 "$pid"
```

Comment:
Hm, this seems to be a way to actually test if the process is still alive which makes sense to test. Maybe this could either be more explicit via `MATCH "sleep" < /proc/$pid/cmdline` or adding a comment.

Reply:
np, added again that check